packages/block-library/src/embed/embed-preview.js

```diff
@@ -28,10 +28,11 @@ const EmbedPreview = ( props ) => {
 	const { scripts } = preview;
 
 	const html = 'photo' === type ? getPhotoHtml( preview ) : preview.html;
-	const parsedUrl = parse( url );
-	const cannotPreview = includes( HOSTS_NO_PREVIEWS, parsedUrl.host.replace( /^www\./, '' ) );
+	const parsedHost = parse( url ).host.split( '.' );
+	const parsedHostBaseUrl = parsedHost.splice( parsedHost.length - 2, parsedHost.length - 1 ).join( '.' );
```

Comment:
It's very non-obvious to me as a future maintainer what it is we're trying to accomplish here with this logic. It would have been very helpful to include one or more of an inline code comment, separate named and documented function, and unit test.